aspnetcore/client-side/libman/samples/LibManSample/LibManSample.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TypeScriptToolsVersion>3.0</TypeScriptToolsVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.1" />
+    <!-- <snippet_RestoreOnBuildPackage> -->
+    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="1.0.113" />
+    <!-- </snippet_RestoreOnBuildPackage> -->
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="wwwroot\lib\" />
```

Comment:
Wouldn't this be taken care of by the Web SDK?

Reply:
Yes. I'll delete it.